torch/csrc/distributed/c10d/init.cpp

```diff
@@ -972,17 +972,44 @@ that adds a prefix to each key inserted to the store.
     py::call_guard<py::gil_scoped_release>());
 #endif
 
+#define PROCESS_GROUP_DEPRECATION_WARNING(api_method)                \
+  TORCH_WARN_ONCE(#api_method                                        \
+                  "API is being deprecated, please ping "            \
```

Comment:
Curious, do you need a space before "API"?

Reply:
fixed